spec/models/similar_occupation_standards_spec.rb

```diff
@@ -0,0 +1,18 @@
+require "rails_helper"
+
+RSpec.describe SimilarOccupationStandards do
+  describe ".similar_to", :elasticsearch do
+    it "returns records similar to the provided, sorted by similarity score" do
+      create(:occupation_standard, title: "Mechanic")
+      medical_assistant_standard = create(:occupation_standard, title: "Medical Assistant")
+      create(:occupation_standard, title: "Medical Assistant")
+      create(:occupation_standard, title: "Admin Assistant")
+
+      OccupationStandard.import
+      sleep 1
+      similars_to_medical_assistant = SimilarOccupationStandards.similar_to(medical_assistant_standard)
+
+      expect(similars_to_medical_assistant.pluck(:title)).to eq ["Admin Assistant", "Medical Assistant"]
```

Comment:
This should actually be the other way around, but IDK why "Admin Assistant" is considered higher relevance by ES.

Reply:
Fixed by adding a snowball analyzer to the query.